### PR TITLE
Sync opam dependencies for xs-extra/ from source repos

### PR DESCRIPTION
**Pull request by GeorgeBurac**
Add xapi-inventory as to xapi-consts.master 
and updated: varstored-guard.master/opam, xapi-consts.master/opam, xapi-database.master/opam, xapi.master/opam, xenctrl.master/opam

Signed-off-by: Gheorghe Burac <gheorghe.burac@citrix.com>

### CODE REVIEW

**Comment by lindig** [comment]
Please update the commit message:

* It needs a more concise subject (long lines don't work well in `git log`)
* A body that explains the why the change is required

File: packages/xs-extra/xapi-consts.master/opam [hunk 1]
```diff
@@ -9,11 +9,12 @@ build: [[ "dune" "build" "-p" name ]]
...
+  "xapi-inventory"
 ]
 synopsis: "The xapi toolstack daemon which implements the XenAPI"
 description: """
 This daemon exposes the XenAPI and is used by clients such as 'xe'
 and 'XenCenter' to manage clusters of Xen-enabled hosts."""
 url {
   src: "https://github.com/xapi-project/xen-api/archive/master/master.tar.gz"
-}
+}
```
Comment (lindig) [inline packages/xs-extra/xapi-consts.master/opam:20]: Newline is missing

**Comment by psafont** [comment]
Please run `tools/generate-opam-sources.sh`, it's not the only out-of-date file.

(Do not update `xen-api-client.master/opam` nor `xenctrl.master/opam`, they are out-of-date in their respective repositories)

File: packages/xs-extra/xenctrl.master/opam [hunk 2]
```diff
@@ -3,8 +3,8 @@ synopsis: "Mock OCaml bindings for the Xen Hypervisor"
 maintainer: "lindig@gmail.com"
 authors: "lindig@gmail.com"
 license: "LGPL"
-homepage: "https://github.com/xapi-project/xenctrl"
```
Comment (psafont) [inline packages/xs-extra/xenctrl.master/opam:6]: Can you leave this file as it is?

**Pull request merged**